test/aspects/ec2-automated-shutdown/index.test.ts

```diff
@@ -0,0 +1,157 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Aspects, Stack } from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import {
+    Instance,
+    InstanceType,
+    InstanceClass,
+    InstanceSize,
+    AmazonLinuxImage,
+    Vpc,
+    SubnetType,
+    SecurityGroup,
+    FlowLogDestination,
+    FlowLogTrafficType,
+    BlockDeviceVolume
+} from 'aws-cdk-lib/aws-ec2';
+import { Key } from 'aws-cdk-lib/aws-kms';
+import { NagSuppressions } from 'cdk-nag';
+import { Ec2AutomatedShutdown } from '../../../src/aspects/ec2-automated-shutdown';
+import { describeCdkTest } from '../../../utilities/cdk-nag-jest';
+
+describeCdkTest(Ec2AutomatedShutdown, (_, getStack, getTemplate) => {
+    let stack: Stack;
+    let template: Template;
+
+    beforeEach(() => {
+        stack = getStack();
+
+        NagSuppressions.addStackSuppressions(stack, [
+            {
+                id: 'AwsSolutions-IAM4',
+                reason: 'Not testing IAM in this test scenario.'
+            },
+            {
+                id: 'AwsSolutions-IAM5',
+                reason: 'Permissions are tightly scoped for EC2 shutdown functionality.'
+            },
+            {
+                id: 'AwsSolutions-S1',
+                reason: 'This S3 bucket is just used for flow logs for the VPC testing the Aspect.'
+            },
+            {
+                id: 'AwsSolutions-S10',
+                reason: 'This S3 bucket is just used for flow logs for the VPC testing the Aspect.'
+            },
+            {
+                id: 'NIST.800.53.R5-S3BucketLoggingEnabled',
+                reason: 'This S3 bucket is just used for flow logs for the VPC testing the Aspect.'
+            },
+            {
+                id: 'NIST.800.53.R5-S3BucketPublicReadProhibited',
```

Comment:
If possible, if we can just set these settings as much as we can on the test constructs we are creating so that we don't have to do these suppressions. If not possible that is OK, but I think some of these would be a simple property add. LMK